README.md

```diff
@@ -25,6 +25,66 @@ Then [connect the adapter](http://sailsjs.com/documentation/reference/configurat
 
 Visit [Models & ORM](http://sailsjs.com/docs/concepts/models-and-orm) in the docs for more information about using models, datastores, and adapters in your app/microservice.
 
+## MongoDB Driver
+From `sails-mongo` version 1.3.0 and above, the adapter uses [MongoDB driver for Node.js v3.5.9 (or above)](https://www.npmjs.com/package/mongodb).
+The updated MongoDB driver changes the way it handles connections internally, and implements the concept of [MongoClient].
+
+`manager` still returns a `database`. Access to the [MongoClient] object is done via `manager.client`:
+```javascript
+// Returns a MongoClient instance
+Pet.getDatastore().manager.client
+```
+
+With access to the [MongoClient] object, now you have access to the latest MongoDB improvements, like [ClientSession],
+and with it, transactions, [change streams](https://mongodb.github.io/node-mongodb-native/3.5/api/ChangeStream.html), and other new features.
+
+#### `.native` still works but you can better use the client
```

Comment:
I think it would make sense to move the example from this section into the [`.manager` documentation](https://sailsjs.com/documentation/reference/waterline-orm/datastores/manager) (and probably not even mention `.native()`, since we say in the docs it's deprecated as of sails v1.x)

Reply:
Actually, made a PR adding it to the tutorial section since that's where we talk about low-level usage: https://github.com/balderdashy/sails/pull/7074